### ISSUE TITLE
Regarding ADNI dataset loader

### ISSUE THREAD

**Issue by LUYU0004** [cyclomon/3dbraingen#3]
Hi,

- May i know have you normalized the ADNI images to [0,1] before loading by `class ADNIdataset`?
As you mentioned in the paper, all inputs shall be normalized to [-1,1]. However, there is no code for normalization as shown in the code snippet from the `class ADNIdataset`.

- Why `np.swapaxes(img.get_data(),1,2)` `np.flip(img, 1)` `np.flip(img,2)` are performed? Those are not conducted in the other dataset loader.

Thank you very much!

```python
		img = nib.load(os.path.join(path,'image.nii'))

		img = np.swapaxes(img.get_data(),1,2)
		img = np.flip(img,1)
		img = np.flip(img,2)

		sp_size = 64
		img = resize(img, (sp_size,sp_size,sp_size), mode='constant')
		if self.augmentation:
			random_n = torch.rand(1)
			random_i = 0.3*torch.rand(1)[0]+0.7
			if random_n[0] > 0.5:
				img = np.flip(img,0)
                
			img = img*random_i.data.cpu().numpy()
            
		imageout = torch.from_numpy(img).float().view(1,sp_size,sp_size,sp_size)
		imageout = imageout*2-1

		return imageout	
```

**Comment by cyclomon** [comment]
Hi, thank you for the comment.

In python 0.4.1 (maybe in the other versions), if the numpy data is changed into float by `torch.from_numpy` function, 
the output tensors are within the range of [0,1].
Thus, I just included `imageout*2-1` to make them into [-1,1].

I just changed the x,y,z order into the form that is familiar to me.
It's just for a good visualization, and also to make the x,y,z order to be same as that of BRATS/ATLAS datasets.
There is no necessity to change the order. 

Thanks!
